### PR TITLE
clean pull embedding vector rpc in Python

### PR DESCRIPTION
**Pull request by QiJune**

### CODE REVIEW

File: elasticdl/python/worker/worker.py [hunk 1]
```diff
@@ -32,9 +32,11 @@
     Tensor,
     emplace_tensor_pb_from_ndarray,
     serialize_tensor,
-    tensor_pb_to_ndarray,
 )
-from elasticdl.python.common.tensor_utils import deduplicate_indexed_slices
+from elasticdl.python.common.tensor_utils import (
+    deduplicate_indexed_slices,
```
Comment (brightcoder01) [inline elasticdl/python/worker/worker.py:37]: I didn't see the usage of deduplicate_indexed_slices in this file

Comment (QiJune) [reply elasticdl/python/worker/worker.py:37]: The `deduplicate_indexed_slices ` is used at 

https://github.com/sql-machine-learning/elasticdl/blob/0a6858da862f6d3f15f41b2c4693078972f848c4/elasticdl/python/worker/worker.py#L513-L519

File: elasticdl/python/ps/servicer.py [hunk 2]
```diff
@@ -78,16 +79,15 @@ def pull_variable(self, request, _):
         res.model_init_status = True
         return res
 
-    def pull_embedding_vector(self, request, _):
-        ret = elasticdl_pb2.Tensor()
+    def pull_embedding_table(self, request, _):
+        res = tensor_pb2.TensorProto()
```
Comment (workingloong) [inline elasticdl/python/ps/servicer.py]: Maybe  "res" -> "result" ?

File: elasticdl/python/worker/worker.py [hunk 3]
```diff
@@ -345,7 +347,7 @@ def get_model(self):
         self._model_version = max(self._model_versions_from_ps)
         self._timing.end_record_time("get_model")
 
-    def pull_embedding_vector(self, layer_name, embedding_ids):
+    def pull_embedding_table(self, layer_name, embedding_ids):
         """Pulls and returns embedding vectors ordered by the embedding ids."""
```
Comment (workingloong) [inline elasticdl/python/worker/worker.py:351]: Why do you rename `pull_embedding_table`? The `pull_embedding_vector` is more suitable for the docstring.

File: elasticdl/python/worker/worker.py (hunk 1 above)
Comment (brightcoder01) [reply elasticdl/python/worker/worker.py]: I'm curious why the import is added in the change, but we can't see the usage in the new added change.
If the two methods are already used in the worker.py, they should be already imported.

File: elasticdl/python/ps/servicer.py (hunk 2 above)
Comment (QiJune) [reply elasticdl/python/ps/servicer.py]: Done

File: elasticdl/python/worker/worker.py (hunk 3 above)
Comment (QiJune) [reply elasticdl/python/worker/worker.py:351]: It's actually pulling some embedding vectors from an embedding table. So maybe `pull_embedding_vectors` is more accurate.
Rename it.

**Review by skydoorkai — APPROVED** [review]

**Pull request merged**